### ISSUE TITLE
iOS Demo build error

### ISSUE THREAD

**Issue by brownkevg** [EddyVerbruggen/nativescript-plugin-firebase#1375]
Building iOS demo app fails with error
**My local env**
nativescript 6.0.3
pod 1.7.5(also tried 1.5.4)
node 10.14.1
**Steps to reproduce**
Clone repo
cd src
npm run setupandinstall
npm run demo.ios
build failed with

`Command Ld failed with a nonzero exit code
note: Using new build systemnote: Planning buildnote: Constructing build description
** BUILD FAILED **
Unable to apply changes on device: 3EE692F2-3D48-4591-9966-41B51ED1ED78. Error is: Command xcodebuild failed with exit code 65.`

Running android works fine... no issues.

I also have this plugin in a project I'm working on and it started to fail on iOS when we upgraded to nativescript 6. When we upgraded it worked fine on android but not iOS. Before we upgraded we were on nativescript 5.4.2 and it worked on both... noticed that after the upgrade a new dependency was installed for GoogleSignIn... went from 4.4.0 to 5.0.0 (https://developers.google.com/identity/sign-in/ios/release). That was the only dependency we noticed that changed. I also noticed that the CI integration for the last two commits failed for iOS with similar errors (https://travis-ci.org/EddyVerbruggen/nativescript-plugin-firebase/builds/575042240?utm_source=github_status&utm_medium=notification)

**Comment by EddyVerbruggen** [comment]
Unfortunately that error is meaningless. Can you share a repo that demonstrates the issue?

**Comment by romandragan** [comment]
@brownkevg, @EddyVerbruggen, my investigation showed that the problem is not with the `GoogleSignIn` itself, but with `remote_config` or `performance_monitoring` options in the `firebase.nativescript.json`.

So it seems like the problem is with `Firebase/RemoteConfig` or `Firebase/Performance` pods.

Futher investigation showed, that both these pods are dependent on `FirebaseABTesting` which has been [updated several days ago to the `3.1.0` version](https://github.com/CocoaPods/Specs/blob/master/Specs/d/c/1/FirebaseABTesting/3.1.0/FirebaseABTesting.podspec.json).

Once I set it to `= 3.0` in the `Podfile` the issue disappeared!

@EddyVerbruggen, temporarily added in my fork with:

```
` + (isSelected(result.remote_config) ? `` : `#`) + `pod 'FirebaseABTesting', '= 3.0'

` + (isSelected(result.performance_monitoring) ? `` : `#`) + `pod 'FirebaseABTesting', '= 3.0'
```

**Comment by brownkevg** [comment]
@EddyVerbruggen Sorry for not clarifying. When I was talking about demo I was referring to nativescript-plugin-firebase's demo https://github.com/EddyVerbruggen/nativescript-plugin-firebase/tree/master/demo

**Comment by EddyVerbruggen** [comment]
@ScottAtRedHawk I’m sorry but that error is useless. Does @romandragan’s suggestion help?

**Comment by EddyVerbruggen** [comment]
@romandragan @ScottAtRedHawk Thanks a bunch! I've added those 2 lines to the script and I don't see build issues. However, when I bump Firebase from 6.6.0 to 6.7.0 this will become an issue again because it depends on a newer versio of the ABTesting pod, and the error resurfaces. But that's for another day..

**Comment by maxorlovsky** [comment]
@EddyVerbruggen it seems like this problem on 9.1.1 is still there. What's interesting it's not happening on every device. 2 out of 5 people have this problem, for the rest everything works fine. But not for demo, for real project. As soon as downgraded to 9.0.4 everything works.

**Comment by EddyVerbruggen** [comment]
@maxorlovsky Then it would be very useful to narrow it down a little further. There must be an explanation for the difference between devices and users..

**Comment by b02505048** [comment]
@EddyVerbruggen I found the problem will still be there on 9.1.1 if enable the following feature of Firebase: 
- Remote Config
- Performance Monitoring
- Crashlytics
- ML Kit

Once I run `npm run config` and set them all to `false`, no more build error on iOS.

Since my project must use ML Kit, therefore I solved the problem by downgrading to 9.0.4 as @maxorlovsky said. And it works charmingly now! Thanks guys!
